onnxruntime/test/testdata/onnx_backend_test_series_filters.jsonc

```diff
@@ -21,8 +21,6 @@
         "^test_nesterov_momentum",
         "^test_pow_types_float32_uint32",
         "^test_pow_types_float32_uint64",
-        "^test_sequence_insert_at_back_cpu", // numpy_helper currently not supporting loading segments.
-        "^test_sequence_insert_at_front_cpu", // numpy_helper currently not supporting loading segments.
```

Comment:
It seems that this one (`test_sequence_insert_at_front_cpu`) is still not working... but it should be another bug which is not related to SequenceProto.

Reply:
Did you consume the fix for the test in ONNX ?

Reply:
ORT with the commit of that bug fix in ONNX (local built ONNX python package) can pass these sequence-related ONNX backend tests now. 

However, updating the ONNX commit to the latest one will bring other issues while building ORT... By contrast, simply cherry-pick this commit https://github.com/onnx/onnx/pull/3136 can be built with ORT without any error. Thus, it seems that something else after ONNX 1.8.0 will break certain ORT test...

Reply:
ORT failure of the latest ONNX issue can be resolved by this PR. https://github.com/onnx/onnx/pull/3175